export fn testThrow()
    throw "error msg"